client/my-sites/preview/main.jsx

```diff
@@ -112,7 +112,7 @@ class PreviewMain extends React.Component {
 
 		return (
 			<Main className="preview">
-				<DocumentHead title={ translate( 'Site Preview' ) } />
+				<DocumentHead title={ translate( 'Your Site' ) } />
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'your site' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).